core/rawdb/accessors_indexes.go

```diff
@@ -123,3 +123,76 @@ func WriteBloomBits(db DatabaseWriter, bit uint, section uint64, head common.Has
 		utils.Logger().Error().Err(err).Msg("Failed to store bloom bits")
 	}
 }
+
+// ReadCxLookupEntry retrieves the positional metadata associated with a transaction
+// hash to allow retrieving the transaction or receipt by hash.
```

Comment:
This only concern the cx receipt, not the original transaction, right? comment needs to be updated.

Reply:
Will add more comment here.